GitUI/Infrastructure/Telemetry/FormBrowseDiagnosticsReporter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using GitCommands;
+using GitUI.CommandsDialogs;
+
+namespace GitUI.Infrastructure.Telemetry
+{
+    internal class FormBrowseDiagnosticsReporter
+    {
+        private readonly FormBrowse _owner;
+
+        public FormBrowseDiagnosticsReporter(FormBrowse owner)
+        {
+            _owner = owner;
+        }
+
+        public void Report()
+        {
+            DiagnosticsClient.TrackEvent($"{_owner.GetType().Name}Start",
+                new Dictionary<string, string>
+                {
+                    // layout
+                    { "ShowLeftPanel", _owner.MainSplitContainer.Panel1Collapsed.ToString() },
+                    { nameof(AppSettings.ShowSplitViewLayout), AppSettings.ShowSplitViewLayout.ToString() },
+                    { nameof(AppSettings.CommitInfoPosition), AppSettings.CommitInfoPosition.ToString() },
+
+                    // revision grid
+                    { nameof(AppSettings.ShowAuthorAvatarColumn), AppSettings.ShowAuthorAvatarColumn.ToString() },
+                    { nameof(AppSettings.ShowAuthorNameColumn), AppSettings.ShowAheadBehindData.ToString() },
```

Comment:
as already mentioned by @mast-eu
```suggestion
                    { nameof(AppSettings.ShowAuthorNameColumn), AppSettings.ShowAuthorNameColumn.ToString() },
```